src/trace-api.js

```diff
@@ -174,15 +173,15 @@ PluginAPI.prototype.runInRootSpan = function(options, fn) {
  * span is created and propogated.
  * @returns A new ChildSpan object, or null if there is no active root span.
  */
-PluginAPI.prototype.createChildSpan = function(options) {
+TraceApiImplementation.prototype.createChildSpan = function(options) {
   var rootSpan = getRootSpan_(this);
   if (rootSpan) {
     options = options || {};
     var childContext = this.agent_.startSpan(options.name, {},
-      options.skipFrames ? options.skipFrames + 1 : 1);
+      options.skipFrames ? options.skipFrames + 2 : 2);
     return new ChildSpan(this.agent_, childContext);
   } else {
-    this.logger_.warn(options.name + ': Attempted to create child span ' +
+    this.logger_.debug(this.pluginName_ + ': Attempted to create child span ' +
```

Comment:
Could we add a `TODO` on this to fix once we can disambiguate context loss from sampling?